### PR TITLE
Fixed cult floors created by Nar-sie not looking like cult floors.

### PR DESCRIPTION
**Pull request by xxalpha**
Fixes https://github.com/tgstation/-tg-station/issues/10180

### CODE REVIEW

**Comment by Cheridan** [comment]
Wasn't #10600 supposed to fix this? It hasn't been merged yet

**Comment by xxalpha** [comment]
Doesn't look like it does. The bug is in the floor code, it actually breaks artificer floor construction too. This fix doesn't fix that, I am testing another fix for both of the issues.

**Comment by Aranclanos** [comment]
really snowflake, go check on changeTurf() of plasteel floors and check if you can make the idiots to not pass the icon if its being replaced with another plasteel.
Something like that

File: code/game/turfs/simulated/floor.dm [hunk 1]
```diff
@@ -165,6 +165,8 @@ var/list/icons_to_ignore_at_floor_init = list("damaged1","damaged2","damaged3","
 
 /turf/simulated/floor/narsie_act()
 	if(prob(20))
+		icon_state = "cult"
+		icon_regular_floor = "cult"
```
Comment (phil235) [inline code/game/turfs/simulated/floor.dm]: this would make cult floor tile always look like cult tile even if you replaced it with a normal floor tile.

Comment (xxalpha) [inline code/game/turfs/simulated/floor.dm]: Working on a better fix.

**Comment by xxalpha** [comment]
untitled

You can do this.

**Comment by Anonus** [comment]
I'd like it if there was a way to get more colored/striped floor tiles but there isn't, and there's no way to tell what tile is what color/stripe.

**Pull request closed without merging**